[FYI] Add FYIs for iOS app submission stuff (#15)
expo/fyi: asc-app-id.md

# App Store Connect app ID

An unique ID for every application existing in App Store Connect. It is typically a 10-digit number.

### How do I find my App ID?

If your application is already on App Store Connect, please follow these steps:

> 1. Open App Store Connect and select your team

Reviewer: ```suggestion
1. Sign in to [App Store Connect](https://appstoreconnect.apple.com/) and select your team
```

> 2. Open the _Apps_ section if it isn't already

Reviewer: ```suggestion
2. Navigate to the [Apps](https://appstoreconnect.apple.com/apps) section if you're not already there
```

3. Click on your app name and ensure that _App Store_ tab is active at the top (next to your app's name)

> 4. On the left pane, under _General_ section, select _App Information

Reviewer: ```suggestion
4. On the left pane, under _General_ section, select _App Information_
```

5. Your App ID can be found under _General Information_ section, called `Apple ID`

>    > Not to be confused with Apple ID as your account username.

Reviewer: ```suggestion
   > Confusingly, _Apple ID_ in this context does not refer to your Apple account username, it refers to your App ID.
```
